src/app/dataset-block/metadata-block/components/event-details/components/common/editor-property/editor-property.component.ts

```diff
@@ -0,0 +1,17 @@
+import { SqlQueryStep } from "./../../../../../../../api/kamu.graphql.interface";
+import { ChangeDetectionStrategy, Component } from "@angular/core";
+import { sqlEditorOptionsForEvents } from "../../../config-editor.events";
+import { BasePropertyComponent } from "../base-property/base-property.component";
+
+@Component({
+    selector: "app-editor-property",
+    templateUrl: "./editor-property.component.html",
+    styleUrls: ["./editor-property.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EditorPropertyComponent extends BasePropertyComponent {
```

Comment:
The name is no longer adequate. We don't use an editor anymore, even a read-only one. Besides, it's specificly showing SQL. So it is an `SqlQueryViewerComponent`